sys-apps/systemd/systemd-256.1-r2.ebuild

```
@@ -180,12 +180,42 @@ BDEPEND="
 QA_FLAGS_IGNORED="usr/lib/systemd/boot/efi/.*"
 QA_EXECSTACK="usr/lib/systemd/boot/efi/*"
 
+check_cgroup_layout() {
+	# https://bugs.gentoo.org/935261
+	[[ ${MERGE_TYPE} != buildonly ]] || return
+	[[ -z ${ROOT} ]] || return
+	[[ -e /sys/fs/cgroup/unified ]] || return
+	grep -q 'SYSTEMD_CGROUP_ENABLE_LEGACY_FORCE=1' /proc/cmdline && return
+
+	eerror "This system appears to be booted with the 'hybrid' cgroup layout."
+	eerror "This layout obsolete and is disabled in systemd."
+
+	if grep -qF 'systemd.legacy_systemd_cgroup_controller' /proc/cmdline; then
+		eerror "Remove the systemd.legacy_systemd_cgroup_controller option"
+		eerror "from the kernel command line and reboot."
+	fi
+
+	if use cgroup-hybrid; then
+		eerror "Disable the 'cgroup-hybrid' USE flag."
+		eerror "Rebuild any initrd images after rebuilding systemd."
```

```suggestion
		eerror "Rebuild any initramfs images after rebuilding systemd."
```
It doesn't really matter but I've been trying to make the usage of `initrd` vs `initramfs` a bit more consistent.

We could also mention here a bit more explicitly what users have to do, i.e. run either `make install` or `emerge --config` again after upgrading this package.

---

Updated initrd to initramfs.

With regard to instructions, I was intentionally vague due to the numerous different ways that people manage their kernel/initramfs.